index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Palasio Quiz</title>
+    <link
+      rel="stylesheet"
+      href="https://palasio-designs.netlify.app/style.css"
+    />
+    <link rel="stylesheet" href="./shared.css" />
+    <link rel="stylesheet" href="./index.css" />
+  </head>
+  <body>
+    <header class="header shadow">
+      <div class="brand">
+        <span class="brand__logo"><i class="bi bi-hurricane"></i></span>
+        <span class="brand__text">PALASIO<br />QUIZ </span>
+      </div>
+
+      <nav>
+        <ul>
+          <li><a href="./login.html">Login</a></li>
+          <li>
+            <a href="/"><btn class="btn primary">Signup</btn></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section>
+        <h1 class="brand-name">PALASIO QUIZ</h1>
+        <h1 class="main__intro-text">
```

Comment:
Are you sure you want both brand name and intro text to be h1?

Reply:
Thank you for pointing that out. I have changed h2 for the second one.